gnucash/gnome/gnc-split-reg.c

```diff
@@ -1089,23 +1107,35 @@ gsr_default_associate_handler_file (GNCSplitReg *gsr, Transaction *trans, gboole
 
     if (response == GTK_RESPONSE_ACCEPT)
     {
+        gboolean add_trailing_slash = FALSE;
         gchar *dialog_uri = gtk_file_chooser_get_uri (GTK_FILE_CHOOSER (dialog));
 
-        PINFO("Dialog File URI: %s\n", dialog_uri);
+        // relative paths do not start with a / so need to add one depending on path_head
```

Comment:
While this is a good catch, I believe it's more efficient to ensure the prefix always has a trailing slash when you save the prefix (wherever that code is stored). Setting a prefix happens less frequently than testing for relative paths against a prefix.